suliman-hello-world.js

```diff
@@ -0,0 +1,21 @@
+//
+// This is only a SKELETON file for the 'Hello World' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+// Make sure to look at test.script.js--that should give you some hints about what is
+// expected here.
+
+var HelloWorld = function() {};
+
+HelloWorld.prototype.hello = function(input) {
+
+  if (typeof(input) === 'undefined') {
+    return "Hello, World!";
+  } else {
+    return 'Hello, ' + input + '!';
+  }
+};
+
+
+
+var test = new HelloWorld();
```

Comment:
That's OK for your tests, but don't forget to restore the right line of code:
`module.exports = HelloWorld;`